packages/trail-hapi-plugin/lib/routes/trails.js

```diff
@@ -49,9 +49,14 @@ module.exports = {
         },
         response: {
           status: {
-            200: Joi.array()
-              .description('The search results.')
-              .items(trailSchema.response),
+            200: Joi.object()
+              .keys({
+                count: Joi.number()
+                  .description('Count of trail records matching filters'),
+                data: Joi.array()
+                  .description('The search results.')
+                  .items(trailSchema.response)
+              }),
```

Comment:
same as earlier comments. let's keep the descriptions consistent between the two. the best way is to extract this object in a schema itself instead of composing it here

Reply:
Adding a new searchResponse object in the schema